packages/devtools_app/lib/src/screens/inspector/layout_explorer/ui/layout_explorer_widget.dart

```diff
@@ -122,7 +122,7 @@ abstract class LayoutExplorerWidgetState<W extends LayoutExplorerWidget,
 
   void updateHighlighted(L? newProperties);
 
-  String? id(RemoteDiagnosticsNode? node) => node?.dartDiagnosticRef.id;
```

Comment:
Make sure to verify that the layout explorer still works after this change. For example, try changing flex values for widgets in a row.

Reply:
https://github.com/flutter/devtools/assets/12115586/3b5362bc-4c50-44f7-a3d3-888037a7a5aa